x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -9,6 +9,8 @@ import { PostureTypes } from './types';
 
 export const STATUS_ROUTE_PATH = '/internal/cloud_security_posture/status';
 export const STATS_ROUTE_PATH = '/internal/cloud_security_posture/stats/{policy_template}';
+export const VULNERABILITIES_STATS_ROUTE_PATH =
```

Comment:
out of curiosity, having our endpoints defined explicitly could simplify our code, what do you think? 
instead of `'/internal/cloud_security_posture/stats/{policy_template}'`

Reply:
I'm not sure, right now we only use it for the `kspm/cspm` functionality in the compliance dashboard (not used in the vulnerabilities dashboard).
We can either say that this route returns data for both integration types by default or copy-paste the logic into two different routes one for each integration type.
Either way, I think that this route is pretty much "done" and not worth investing time to recreate, I also don't see the complicated part that we would like to simplify but that might be because I'm familiar with this code.